### ISSUE TITLE
[Feature] Allow absolute values for metric thresholds

### ISSUE THREAD

**Issue by itsgrimetime** [growthbook/growthbook#1241]
### Description of Feature

Allow absolute values for metric thresholds.

Right now metric thresholds only support % values, we've got a few use cases where folks want to be able to use absolute metric values. These would likely be overridden on specific experiments, but there are some where it would be globally applicable to specific metrics.

![Screenshot 2023-05-01 at 11 09 05 AM](https://user-images.githubusercontent.com/990274/235494817-e50a56b8-8ab4-4a36-a959-b326aac845f4.png)

My thinking is that the "%" on the right side could be a drop down, where "value" would map to an absolute value for the thresholds.

**Comment by lukesonnet** [comment]
Not quite. This setting just controls the front-end results and whether or not we display results when the uplift it outside of some range. So we do have percentile and absolute capping which might have been what you were looking for originally.

I'm going to leave this open now that we are landing absolute differences in the results table #1766
